composer.json

```diff
@@ -70,20 +70,21 @@
         "phpstan/phpstan-phpunit": "^1.1",
         "phpstan/phpstan-webmozart-assert": "^1.2",
         "phpunit/phpunit": "^9.5",
+        "rector/rector": "^0.18.2",
         "sylius-labs/coding-standard": "^4.0",
         "sylius/grid-bundle": "^1.7 || dev-symfony-7",
         "symfony/console": "^5.4 || ^6.4 || ^7.0",
         "symfony/dependency-injection": "^5.4 || ^6.4 || ^7.0",
         "symfony/dotenv": "^5.4 || ^6.4 || ^7.0",
+        "symfony/http-kernel": "^5.4 || ^6.4 || ^7.0",
```

Comment:
normally this kind of things is already ok if we use Docker or CI to set up the dev requirements.
If you don't use docker it will not use the "symfony requirement from Flex" filter.

Reply:
but I think it's better to explicit all our dev dependencies.